components/ILIAS/Link/resources/ilExtLink.js

```diff
@@ -8,11 +24,14 @@ il.ExtLink = {
 	 * Linkify wrapper
 	 */
 	autolink: function (selector, link_class) {
-		$(selector).linkify({
+		const options = {
 			validate: {
 				url: (val) => /^https?:\/\//.test(val), // only allow URLs that begin with a protocol
 				email: false // don't linkify emails
 			}
+		};
+		$(selector).each(function () {
```

Comment:
One last question: Could we easily get rid of the jQuery calls in this file? If yes, it would be nice if we could use vanilla JS.

Reply:
I kept the jQuery call to be fully backwards compatible. As far as I can see `il.ExtLink` is used in the `Chatroom`, `OnScreenChat` and `COPage`. Which need support for a single `HTMLNodeElement` and for a `selector`. If the jQuery call should be replaced I would rather support these 2 cases but nothing more. If this should be fully backwards compatible, this would need support for `HTMLNodeElement`, `selector`, `jQuery Node`, `Arrays`, etc.
Do you think we should drop the unused cases?

Reply:
@alex40724 Any opinion here, cause you are one of the consumers. If you don't have the time to answer in a timely manner, it's okay. We will then leave it as itr is for now.

I personally would like to drop the jQuery support and only support the use cases of the current consumers.